src/components/Class/ClassCard/index.js

```diff
@@ -183,60 +198,137 @@ function ClassCard({ item, editClass, enroll, style }) {
         <Typography
           variant="subtitle1"
           color="primary"
-          className={classes.spacing}
+          sx={{
+            display: "flex",
+            justifyContent: "space-between",
+          }}
         >
           {languageMap[item.type]}
           {item.enrolled && (
             <i className="check-icon check-icon fa fa-check-circle">Enrolled</i>
           )}
+          {((rolesList.length == 0 && item.enrolled) ||
+            (rolesList.length >= 1 &&
+              (item.facilitator.email === user.data.user.email || flag))) && (
+            <MoreVertIcon onClick={handleOpenUserMenu} sx={{ p: 0 }} />
+          )}
         </Typography>
+        <Menu
+          sx={{ mt: "15px" }}
+          id="menu-appbar"
+          anchorEl={anchorElUser}
+          anchorOrigin={{
+            vertical: "top",
+            horizontal: "right",
+          }}
+          keepMounted
+          transformOrigin={{
+            vertical: "top",
+            horizontal: "right",
+          }}
+          open={Boolean(anchorElUser)}
+          onClose={() => {
+            setAnchorElUser(null);
+          }}
+        >
+          {(item.facilitator.email === user.data.user.email || flag) && (
+            <>
+              <MenuItem
+                onClick={() => handleClickOpen(item.id)}
+                sx={{ width: 100, margin: "0px 10px" }}
+              >
+                <Typography textAlign="center">Delete</Typography>
+              </MenuItem>
+              <MenuItem
+                onClick={() => handleEdit(item.id)}
+                sx={{ width: 100, margin: "0px 10px" }}
+              >
+                <Typography textAlign="center">Edit</Typography>
+              </MenuItem>
+            </>
+          )}
+
+          {!rolesList.includes("volunteer") && (
+            <MenuItem
+              onClick={() => handleClickOpenUnenroll(item.id)}
+              sx={{ width: 120, margin: "0px 10px" }}
+            >
+              <Typography textAlign="center">Dropout</Typography>
+            </MenuItem>
+          )}
+        </Menu>
         <Typography variant="subtitle1" className={classes.spacing}>
           {item.title}
         </Typography>
-        <Typography className={classes.spacing}>
-          Facilitator : {item.facilitator.name}
+        <Typography variant="body1" sx={{ display: "flex" }}>
+          <img
+            className={classes.icons}
+            src={require("../assets/calendar.svg")}
+          />
+          {dateTimeFormat(item.start_time).finalDate}
         </Typography>
-        <Typography className={classes.spacing}>
-          Language : {languageMap[item.lang]}
+        <Typography variant="body1" sx={{ display: "flex" }}>
+          <img className={classes.icons} src={require("../assets/time.svg")} />
+          {moment(classStartTime).format("hh:mm a")} -{" "}
+          {moment(classEndTime).format("hh:mm a")}
```

Comment:
As before, if not urgent, we could use the `format` from the date.js file I created in my other PR that would use `date-fns` here.